### PR TITLE
Adding d-voting project

### PR DESCRIPTION
**Pull request by ineiti**
Adds a stub for the d-voting project.

### CODE REVIEW

File: data/DEDIS/projects.yaml [hunk 1]
```diff
@@ -571,3 +571,33 @@ projects:
...
+    description: >
+      (Short description, will be shown under the name)
+    categories:
+      - "Blockchain"
+    applications:
+      - "Gov"
+    type: Application
+    layman_desc: >
+      The d-voting project allows for electronic voting without a central point of trust.
+      Instead it relies on the majority of servers to be honest during the voting process.
```
Comment (PascalinDe) [inline data/DEDIS/projects.yaml:586]:
```suggestion
      It relies instead on the majority of servers to be honest during the voting process.
```

File: views/products/pilot/dvoting.tpl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The d-voting project allows for electronic voting without a central point of trust.
+Instead it relies on the majority of servers to be honest during the voting process.
```
Comment (PascalinDe) [inline views/products/pilot/dvoting.tpl:2]:
```suggestion
It relies instead on the majority of servers to be honest during the voting process.
```

File: views/products/pilot/dvoting.tpl [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+The ballots are encrypted during the voting phase, and then shuffled to anonymise the results.
+When they are finally decrypted, it is not possible anymore to retrace who voted for what.
+
+As of September 2023, we're preparing to launch a demo of the d-voting across several instances of EPFL.
+We hope to replace the current e-voting project for March 2024.
```
Comment (PascalinDe) [inline views/products/pilot/dvoting.tpl:7]:
```suggestion
We plan to replace the current e-voting project for March 2024.
```

File: data/DEDIS/projects.yaml [hunk 4]
```diff
@@ -571,3 +571,33 @@ projects:
     license: GPL-3.0
     date_added: 2023-07-05
     date_updated: 2023-07-05
```
Comment (PascalinDe) [inline data/DEDIS/projects.yaml:573]: shouldn't these dates be changed as well?

Comment (ineiti) [reply data/DEDIS/projects.yaml:573]: Only with the changes from Ahmed.

**Review by PascalinDe — APPROVED** [review]
LGTM

**Pull request merged**